Observer/NonBoltOrderObserver.php

```diff
@@ -197,15 +206,65 @@ protected function getPhone($cart)
             return null;
         }
 
-        if (count($cart['shipments']) < 1) {
-            return null;
+        if (isset($cart['shipments'][0]['shipping_address'])) {
```

Comment:
this condition covers the previous one so that one can be removed

Reply:
oops meant to include that as well